packages/components/text/src/Text.tsx

```diff
@@ -149,6 +149,7 @@ export const Text = compressible<TextProps, TextTokens>((props: TextProps, useTo
       numberOfLines: truncate || !wrap ? 1 : 0,
       style: mergeStyles(tokenStyle, props.style, extra?.style, scaleStyleAdjustments),
     };
+    delete mergedProps.style.dynamicTypeRamp; // TODO(#2268): RN Text doesn't recognize dynamicTypeRamp yet, so don't let it leak through or RN will be sad
```

Comment:
Can you clarify on the "RN will be sad" bit? Do you get a Redbox?

Reply:
It gives a warning about an invalid prop. Updated to be more clear.